app/uk/gov/hmrc/helptosave/controllers/HelpToSaveAPIController.scala

```diff
@@ -25,21 +25,22 @@ import uk.gov.hmrc.helptosave.config.AppConfig
 import uk.gov.hmrc.helptosave.connectors.{HelpToSaveProxyConnector, ITMPEnrolmentConnector}
 import uk.gov.hmrc.helptosave.models.{ErrorResponse, NSIUserInfo}
 import uk.gov.hmrc.helptosave.repo.EnrolmentStore
-import uk.gov.hmrc.helptosave.services.UserCapService
+import uk.gov.hmrc.helptosave.services.{EligibilityCheckService, UserCapService}
 import uk.gov.hmrc.helptosave.util.JsErrorOps._
 import uk.gov.hmrc.helptosave.util.Logging._
-import uk.gov.hmrc.helptosave.util.{LogMessageTransformer, Logging, toFuture}
+import uk.gov.hmrc.helptosave.util.{LogMessageTransformer, toFuture}
 import uk.gov.hmrc.play.bootstrap.controller.BaseController
 
 import scala.util.{Failure, Success}
 
-class CreateDEAccountController @Inject() (val enrolmentStore: EnrolmentStore,
-                                           val itmpConnector:  ITMPEnrolmentConnector,
-                                           proxyConnector:     HelpToSaveProxyConnector,
-                                           userCapService:     UserCapService)(
+class HelpToSaveAPIController @Inject() (val enrolmentStore:          EnrolmentStore,
+                                         val itmpConnector:           ITMPEnrolmentConnector,
+                                         proxyConnector:              HelpToSaveProxyConnector,
+                                         userCapService:              UserCapService,
+                                         val eligibilityCheckService: EligibilityCheckService)(
     implicit
     transformer: LogMessageTransformer, appConfig: AppConfig)
-  extends BaseController with Logging with WithMdcExecutionContext with EnrolmentBehaviour {
+  extends BaseController with EligibilityBase with WithMdcExecutionContext with EnrolmentBehaviour {
 
   def createDEAccount(): Action[AnyContent] = Action.async {
```

Comment:
stride frontend is also calling this endpoint so `ApiController` isn't strictly true - is something like `HelpToSaveNonDigitalController` more accurate?